stream_test.go

```diff
@@ -397,6 +397,21 @@ func TestStream_NextChunkRange(t *testing.T) {
 				end:   defaultStart.Add(7 * defaultDuration / 2),
 			},
 		},
+		{
+			name: "Start/End cannot be the same",
+			s: stream{
+				QuerySpec: QuerySpec{
+					Start: defaultEnd.Add(-3 * time.Hour),
+					End:   defaultEnd,
+					Chunk: time.Minute * 1,
+				},
+				next: 180,
```

Comment:
I think if I'm understanding correctly, this is trying to recreate a state that should not be possible if the rest of the Incite engine is working correctly.

Basically a 3-hour time range chunked in minutes has 180 chunks numbered `0` through `179`. Chunk number `180` is "one past the end" and so you would expect it to have odd characteristics.

The questions in my mind are:

1. Can the rest of the engine get into a bad state where it is trying to create a "one past the end" chunk range? If so, what is the root cause?
2. If not, is there a set of "valid" inputs to `nextChunkRange` that still causes the undesirable behavior?